strax/plugin.py

```diff
@@ -902,7 +902,9 @@ class ParallelSourcePlugin(Plugin):
     @classmethod
     def inline_plugins(cls, components, start_from, log):
         plugins = components.plugins.copy()
-
+        loader_plugins = components.loader_plugins.copy()
```

Comment:
You mention this in the PR description but for my understanding, why could we not add this to the `plugins`-dict  and have to make a separate attribute?

Reply:
Because then it would be recomputed.